processors/sshd/sshdProcessor.go

```diff
@@ -89,7 +137,17 @@ func extraDataWithCA(alg, keySum, certSerial, caData string) (*json.RawMessage,
 	return &rawmsg, err
 }
 
-type processEntryConfig struct {
+func NewSshdProcessor(ctx context.Context, logins chan<- common.RemoteUserLogin, nodeName string, machineID string, eventW *auditevent.EventWriter) *SshdProcessor {
```

Comment:
Created to pass the `Process` func to the ingester. only variables that change are pid and log entry.